packages/yourloops/lib/language.ts

```diff
@@ -101,10 +102,7 @@ i18n.on('languageChanged', (lng: string) => {
   }
 });
 
-async function initI18n(): Promise<TFunction> {
-  return i18n.init(i18nOptions);
-}
-
+i18n.init(i18nOptions);
```

Comment:
The init function is a promise, may be async, should be taken into account

Reply:
This is why we need the Suspense wrapper around the app

Reply:
This page describes experimental features that are not yet available in a stable release.
https://reactjs.org/docs/concurrent-mode-suspense.html

Reply:
This is the solution given by [react.i18n](https://react.i18next.com/latest/using-with-hooks). Should we find another solution ?